### ISSUE TITLE
Improve quickpick

### ISSUE THREAD

**Issue by ChristianKohler** [ChristianKohler/NpmIntellisense#29]
This is thread to collect ideas on how quickpick can be improved.

Some ideas:
- Use default export variable when default export is defined. Otherwise named export
- Place cursor into named export braces or select default export variable after adding the statement

**Comment by dgieselaar** [comment]
It'll probably be a bit difficult to get the exports for a module. The file might contain flow/typescript/jsx etc which makes it difficult for NpmIntellisense to parse. A solution might be to look for babel locally and parse the file.

**Comment by dgieselaar** [comment]
Another thing, my implementation also included local modules in addition to those in `node_modules`.

**Comment by ChristianKohler** [comment]
I added support for local modules with 1.1.0 but forgot to update the documentation. Fixed that with 1.2.1. See: https://github.com/ChristianKohler/NpmIntellisense#show-build-in-local-libs

**Comment by dgieselaar** [comment]
Sorry for the late reaction! What I meant was local (or relative) modules/files such as ./app/foo/index.js, not built-in modules like fs and path. Do you have plans to support that?

**Comment by yaas-dev** [comment]
@dgieselaar is this what you are looking for?
I believe there is already an extension for that: https://github.com/ChristianKohler/PathIntellisense

**Comment by dgieselaar** [comment]
@yaas-dev yeah, maybe! This is pretty long ago, I'm not sure what existed at that point. I'm not using the extension anymore either. Appreciate the update though!
